plugins/modules/zos_fetch.py

```diff
@@ -91,27 +88,27 @@
     suboptions:
       from:
         description:
-            - The encoding to be converted from
+            - The encoding to be converted from.
         required: true
         type: str
       to:
         description:
-            - The encoding to be converted to
+            - The encoding to be converted to.
         required: true
         type: str
 notes:
     - When fetching PDS(E) and VSAM data sets, temporary storage will be used
       on the remote z/OS system. After the PDS(E) or VSAM data set is
-      successfully transferred, the temporary data set will deleted. The size
+      successfully transferred, the temporary data set will be deleted. The size
       of the temporary storage will correspond to the size of PDS(E) or VSAM
       data set being fetched. If module execution fails, the temporary
       storage will be cleaned.
     - To prevent redundancy, additional checksum validation will not be done
       when fetching PDS(E) because data integrity checks are done through the
```

Comment:
when fetching a PS(sequential data set), PDS, PDSE, member of a PDS/PDSE or
KSDS(VSAM data set). 

@asifmahmud  ^^^ is the above true.

Reply:
@ddimatos The above is true only when fetching partitioned data sets (PDS/PDSE) and VSAM.